.github/release/full/Dockerfile

```diff
@@ -34,9 +34,12 @@ RUN pip install mysqlclient
 # 2. Main
 FROM python:3.8.8-slim
 
+ENV DEBIAN_FRONTEND noninteractive
+ENV TERM linux
+
 # Install libgdal.
 RUN apt-get update
-RUN apt-get --yes install libgdal20
+RUN apt-get --yes install libgdal20 git
```

Comment:
I think this should go with # Install GDAL (line 20) + git is already installed there

Reply:
It is a multi-stage Dockerfile. At line 34, the outcome Docker image is produced. So, `git` would not have been part of the image.

However, I only needed it for installing a development version of [1], so it can be removed again.

[1] https://github.com/earthobservations/filesystem_spec/tree/integration